doc/_build/html/_modules/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```

Comment:
We do not need to include the [doc/_build/](https://github.com/ljwolf/spreg/tree/master/doc/_build) folder in the repo as readthedocs will read conf.py and relevant source rst code, and build the html files for us. Committing this folder (and all the files inside) would make the package larger.

Reply:
gotcha; have done so.